examples/imdb_fasttext.py

```diff
@@ -21,7 +22,49 @@
 from keras.datasets import imdb
 
 
+def create_ngram_set(input_list, ngram_range=2):
+    """
+    Extract n-grams from a list of integers
+
+    >>> create_ngram_set([1, 4, 9, 4, 1], ngram_range=2)
+    {(4, 9), (4, 1), (1, 4), (9, 4)}
+
+    :param input_list:
+    :param ngram_range: integer
+    :return: set of tuples
+    """
+    return set(zip(*[input_list[i:] for i in range(ngram_range)]))
```

Comment:
The `create_ngram_set` function is used for creating `token_indice` dictionary which maps a unique token (`(1,5)` bi-gram for instance) to a unique integer value. That is why I need tu use `set` instead of `list`.

AFAIK, the ordering of each integer sequence does not matter, thus I only need to add new integers that correspond to n-gram interactions.